src/docs/cookbook/persistence/reading-writing-files.md

```diff
@@ -36,9 +36,8 @@ device's file system. The plugin currently supports access to
 two file system locations:
 
   * *Temporary directory:* A temporary directory (cache) that the system can
-    clear at any time. On iOS, this corresponds to the value that
-    [`NSTemporaryDirectory()`](https://developer.apple.com/reference/foundation/1409211-nstemporarydirectory)
-    returns. On Android, this is the value that
+    clear at any time. On iOS, this corresponds to the
+    [`NSCachesDirectory`](https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nscachesdirectory?language=occ). On Android, this is the value that
```

Comment:
Would you mind updating the link? You don't need that bit at the end. Instead, use:

https://developer.apple.com/documentation/foundation/nssearchpathdirectory/nscachesdirectory

Reply:
Thank you. Fixed.